scripts/generate_psa_constants.py

```diff
@@ -179,6 +186,12 @@
 '''
 
 class MacroCollector:
+    """Collect PSA crypto macro definitions from C header files.
```

Comment:
Strange indentation

Reply:
Is it? What's strange about it?

Reply:
The ending and beginning quotes are misaligned, and the comments from lines below start at the beginning of the line.

Reply:
That's one of the valid indentation styles for docstrings. But I notice that other parts of the script use a different style, so I'll update this one to be consistent.